doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc

```diff
@@ -99,3 +98,9 @@ AWS provides a https://github.com/awslabs/aws-advanced-jdbc-wrapper[JDBC driver
 To disable the AWS JDBC driver, set the `KC_USE_AWS_JDBC_WRAPPER` variable to `false`.
 
 To specify the version of the AWS JDBC driver, set the `KC_AWS_JDBC_WRAPPER_URL` variable to the URL of corresponding jar file.
+
+=== Warnings / Known issues
+
+* Using `--force` will break the keycloak deployment because it patches the keycloak operator deployment with an incorrect image.
```

Comment:
I believe this is no longer a problem as we are not patching operator deployment when setting the Keycloak image.